Automattic/jetpack
# Typography: first pass at implementing a consistent type scale

### Changes proposed in this Pull Request:

This PR contains a first pass at implementing a consistent type scale across the entire plugin that matches what we use in Calypso. This type scale was inherited from Core, so the hope is that we can converge all 3 platforms with a consistent, harmonic typography.

As a rule of thumb to improve legibility, if the existing `font-size` was smaller than what is specified in the type scale — e.g.: 13px vs `rem( 14px )`, the `font-size` value was increased to the closest value available.

More details on p6TEKc-3Nh-p2

Here's a quick breakdown:

* Update typography partial with [Calypso's type scale](https://wpcalypso.wordpress.com/devdocs/typography).
* Updated main CSS file.
* Updated typography in all main pages:
  * At a Glance
  * My Plan
  * Plans
  * Settings
* Updated typography in components.
* Removed a duplicate `rem()` function file and updated all instances.

### Jetpack product discussion

More details on p6TEKc-3Nh-p2

### Testing instructions:

* Go to wp-admin.
* Browse around the entire plugin.
* Confirm that everything looks good and typography rules are updated.

### Proposed changelog entry for your changes:

* Improved legibility across all Jetpack interfaces.

### Screenshots

Before | After
--|--
![image](https://user-images.githubusercontent.com/390760/88403289-44379980-cdc4-11ea-9570-282c880e79b7.png) | ![image](https://user-images.githubusercontent.com/390760/88403310-4a2d7a80-cdc4-11ea-9a63-1b0af9b06e30.png)
![image](https://user-images.githubusercontent.com/390760/88403391-66311c00-cdc4-11ea-8b4d-3e29e12e430b.png) | ![image](https://user-images.githubusercontent.com/390760/88403418-70531a80-cdc4-11ea-87aa-1f93bb6022b6.png)
![image](https://user-images.githubusercontent.com/390760/88403521-9678ba80-cdc4-11ea-9dce-71f29bd8ec4a.png) | ![image](https://user-images.githubusercontent.com/390760/88403556-a09ab900-cdc4-11ea-948a-8a7fece31370.png)
![image](https://user-images.githubusercontent.com/390760/88403638-b5774c80-cdc4-11ea-8ab9-150b231e0497.png) | ![image](https://user-images.githubusercontent.com/390760/88403690-c2943b80-cdc4-11ea-9d20-c6eed4af5548.png)
